engine/src/main/java/org/terasology/config/facade/InputDeviceConfiguration.java

```diff
@@ -3,21 +3,38 @@
 
 import org.terasology.config.Config;
 import org.terasology.config.ControllerConfig.ControllerInfo;
-import org.terasology.context.Context;
 
 /**
  * Facade for {@link Config#getInput()}
  */
 public interface InputDeviceConfiguration {
+    /**
+     * Returns the mouse sensitivity in the range of [0,1]
+     */
     float getMouseSensitivity();
 
+    /**
+     * Sets the mouse sensitivity, the value should be in [0,1].
+     */
     void setMouseSensitivity(float mouseSensitivity);
 
-    void reset(Context context);
+    /**
+     * Resets the the mouse sensitivity and axis settings from the default configuration.
```

Comment:
`* Resets the the mouse sensitivity and axis settings back to the default values.`